### PR TITLE
Post-Launch ETL

### PR DESCRIPTION
**Pull request by lstod**
- left dummy commands for application and noi
- migrate.py is used for post-launch
- pre-launch migrate.py has been changed to migrate_initial
- updated readme to reflect changes

### CODE REVIEW

File: bin/migrate-oats-data/etl_log.txt
Comment (mhuseinov) [inline bin/migrate-oats-data/etl_log.txt:1]: I think this is a redundant file

Comment (lstod) [reply bin/migrate-oats-data/etl_log.txt:1]: yep looks like it

**Comment by lstod** [comment]
yes that it correct, just left them there to keep the structure.

**Review by trslater — APPROVED** [review]
LGTM 🚀

**Pull request merged**